ERCS/erc-7579.md

```diff
@@ -223,6 +229,9 @@ If the smart account has a fallback handler installed, it:
 - MUST route to fallback handlers based on the function selector of the calldata
 - MAY implement authorization control, which SHOULD be done via hooks 
 
+If the account is adding features via fallback, in context of requirements defined by this specification, it should be considered the same as if the account was implementing those features natively.
+ERC-165 support (see below) is one example of such an approach.
```

Comment:
```suggestion
ERC-165 support (see below) is one example of such an approach. Note, that it is only RECOMMENDED to implement view functions via fallback where this can lead to greater extensibility. It is NOT RECOMMENDED to implement core account logic via a fallback.
```